src/main/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiver.java

```diff
@@ -64,6 +64,10 @@ public void receiveMessage(FieldworkFollowup followup) {
       actionRequest.setCeExpectedResponses(Integer.parseInt(followup.getCeExpectedCapacity()));
     }
 
+    actionRequest.setUndeliveredAsAddress(false);
```

Comment:
it feels like we should be hardcoding these values in the action-schedule as we're going to need to implement them there eventually anyway

Reply:
Good shout, addressed here and in https://github.com/ONSdigital/census-rm-action-scheduler/pull/24 👍